src/slurmmail/cli.py

```diff
@@ -891,6 +898,11 @@ def send_mail_main():
         options.email_from_name = config.get(section, "emailFromName")
         options.email_subject = config.get(section, "emailSubject")
 
+        if config.has_option(section, "emailHeaders"):
+            email_headers = [ x.strip().split(":", maxsplit=1) for x in config.get(section, "emailHeaders").split(";") ]
```

Comment:
To popuate the `options.email_headers` dictionary, two loops are being used here, one to create `email_headers` and then another to iterate through this list. Instead this can be accomplished with one loop which is slightly easier to read also:

```python
for header in config.get(section, "emailHeaders").split(";"):
        header_name, header_value = header.split(":", maxsplit=1)
        email_headers_dict[header_name.strip()] = header_value.strip()
```

Reply:
Adjusted based on the feedback